lib/resources/postgres.rb

```diff
@@ -3,6 +3,7 @@
 # author: Dominik Richter
 # author: Christoph Hartmann
 # license: All rights reserved
+require 'resources/directory'
```

Comment:
we do not need a require here, since its available by default via `inspec.directory`